.eslintrc.json

```
@@ -0,0 +1,38 @@
+{
+    "env": {
+        "browser": true,
+        "es6": false,
+        "jquery": true,
+        "greasemonkey": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:vue/essential",
+        "plugin:es5/no-es2016"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "vue",
+        "es5"
+    ],
+    "rules": {
+        "no-undef": 0,
+        "eqeqeq": "error",
+        "indent": ["warn", 2],
+        "brace-style": "error",
+        "keyword-spacing": ["error", { "before": true }],
```

```suggestion
        "keyword-spacing": "error",
```
No need in `before` as it by default.

---

done